src/Microsoft.Extensions.Configuration.EnvironmentVariables/project.json

```diff
@@ -6,7 +6,8 @@
     "nowarn": [ "CS1591" ],
     "xmlDoc": true
   },
-  "description": "Environment variables configuration provider for the Configuration package.",
+  "description": "Environment variables configuration provider implementation of the Microsoft.Extensions.Configuration.IConfigurationProvider interface.",
```

Comment:
I get that mentioning the interface here makes it very unambiguous what the package is,  but for most people building and consuming configuration (as opposed to people writing providers) the interface means (and should mean) nothing. What about using the namespace instead?